packages/create-remix/templates/vercel/remix.config.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @type {import('@remix-run/dev/config').AppConfig}
+ */
+module.exports = {
+  appDirectory: "app",
+  browserBuildDirectory: "public/build",
+  publicPath: "/build/",
+  serverBuildDirectory: "api/build",
+  devServerPort: process.env.PORT || 3000
```

Comment:
[`devServerPort`](https://discord.com/channels/770287896669978684/771068344320786452/912549069711360101) is actually the port for `WebSocket` used in `<LiveReload />`

Reply:
Got it, thank you!

I removed it in https://github.com/remix-run/remix/pull/588/commits/1fb9798bece5b84f7b67528a1f1fdb8bb0e6670b, because @mcansh mentioned that the dev server accepts the Environment Variable by default anyways.